djreact/djangoapps/api/paginators.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+from rest_framework.response import Response
+from rest_framework.pagination import PageNumberPagination
+
+
+class CustomResponsePagination(PageNumberPagination):
+    """
+    API Custom paginator to tell number of pages
+    on which data has been distributed.
+    """
+
+    page_size = 10
+
+    def get_paginated_response(self, data):
```

Comment:
why are we overriding this method?

Reply:
the default pagination returns only:
1. next
2. prev
3. count
4. data

I made the custom paginator to add total page and current page information so that our pagination is more flexible and we can easily navigate b/w the results(from the front-end perspective)